packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -29,14 +39,28 @@
 @SuppressWarnings("deprecation")
 public class LocalAuthPlugin implements MethodCallHandler, FlutterPlugin, ActivityAware {
   private static final String CHANNEL_NAME = "plugins.flutter.io/local_auth";
+  private static final int LOCK_REQUEST_CODE = 221;
+
+  private ActivityPluginBinding activityPluginBinding;
+  private Registrar registrar;
+
+  private Activity getActivity() {
+    if (activityPluginBinding != null) {
+      return activityPluginBinding.getActivity();
+    }
+    if (registrar != null) {
+      return registrar.activity();
+    }
+    return null;
```

Comment:
Should  you ever be here? I don't see you checking  for null in everywhere getActivity is called  below.